projects/power_select/inc/power_select_defs.h

```diff
@@ -26,22 +27,31 @@
 #define POWER_SELECT_AUX_MAX_CURRENT_MA 8220
 #define POWER_SELECT_AUX_MAX_VOLTAGE_MV 15820
 
+// Cell needs to be above 2V to work, so setting cutoff at 2100 mV
+#define POWER_SELECT_CELL_MIN_VOLTAGE_MV 2100
+
 // Offsets for use in fault bitset
 typedef enum {
-  POWER_SELECT_AUX_OVERVOLTAGE = 0,
-  POWER_SELECT_DCDC_OVERVOLTAGE,
-  POWER_SELECT_PWR_SUP_OVERVOLTAGE,
-  POWER_SELECT_AUX_OVERCURRENT,
-  POWER_SELECT_DCDC_OVERCURRENT,
-  POWER_SELECT_PWR_SUP_OVERCURRENT,
-  POWER_SELECT_DCDC_FAULT,
+  POWER_SELECT_FAULT_AUX_OV = 0,

```

Comment:
Though I like that this naming is shorter. I am worried someone will come across this and see OV or OC and get confused.
Could we add this to the confluence + maybe a comment somewhere? 

Alternatively, We could keep it as OVERVOLTAGE or OVERCURRENT

Reply:
Yeah, OV/OC is probably a bit obfuscated.  With OVERVOLTAGE/OVERCURRENT they were really really long and were causing code to get broken across multiple lines, so I'll add a comment + mention it in the confluence.